client/filestore-client/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@raincatcher/filestore-client",
+  "version": "1.0.0",
+  "description": "RainCatcher file storage client",
+  "types": "src/index.ts",
+  "author": "feedhenry-raincatcher@redhat.com",
+  "license": "Apache-2.0",
+  "main": "src/",
+  "scripts": {
+    "clean": "del coverage_report src/**/*.js src/**/*.map test/**/*.js test/**/*.map",
+    "build": "tsc",
+    "start": "ts-node src/index.ts",
```

Comment:
No need for a `start` command on this package, maybe we should remove it from the base?